modules/performanceplatform/manifests/proxy_vhost.pp

```diff
@@ -53,6 +53,13 @@
 
   $graphite_servername = regsubst($servername, '\.', '_', 'G')
 
+  if ($serveraliases != undef) {
+    validate_array($serveraliases)
+    $vhost_names = $serveraliases
+  } else {
+    $vhost_names = [$servername]
```

Comment:
Not sure I follow. If it's defined, it should be an array.

If it's not defined, it will explicitly do the default that the nginx module already does (and the existing behaviour prior to this addition). If I add the diff below, this is how it should be used.

```diff
diff --git a/hieradata/role-development.yaml b/hieradata/role-development.yaml
index e56970c..6ed6d96 100644
--- a/hieradata/role-development.yaml
+++ b/hieradata/role-development.yaml
@@ -75,6 +75,9 @@ vhost_proxies:
   # For spotlight
   spotlight:
     servername:    "%{::spotlight_vhost}"
+    serveraliases:
+      - "%{::spotlight_vhost}"
+      - "jabley.%{::spotlight_vhost}"
     ssl:           true
     upstream_port: 3057
   # For backdrop
```